tests/integration/test_akafka_testutils.py

```diff
@@ -15,21 +15,50 @@
 #
 
 """Testing of the Kafka testutils."""
-
 from typing import Sequence
 
 import pytest
+from kafka import KafkaAdminClient
 
 from hexkit.providers.akafka import KafkaConfig, KafkaEventPublisher
-from hexkit.providers.akafka.testutils import kafka_fixture  # noqa: F401
-from hexkit.providers.akafka.testutils import (
+from hexkit.providers.testing.fixtures import kafka_fixture  # noqa: F401
+from hexkit.providers.testing.fixtures import (
     ExpectedEvent,
     KafkaFixture,
     RecordedEvent,
     ValidationError,
 )
 
 
+def test_delete_topics_specific(kafka_fixture: KafkaFixture):  # noqa: F811
+    """Make sure the reset function works"""
+    admin_client = KafkaAdminClient(
+        bootstrap_servers=kafka_fixture.config.kafka_servers
+    )
+
+    initial_topics = admin_client.list_topics()
+    initial_length = len(initial_topics)
+
+    assert initial_length > 0
+
+    kafka_fixture.delete_topics(initial_topics)
+
+    assert len(admin_client.list_topics()) + 1 == initial_length
```

Comment:
So there is some topic initially that's not returned and thus deleted?
Not sure I quite get what is happening here

Reply:
In [044edc8](https://github.com/ghga-de/hexkit/pull/59/commits/044edc8a8721eb6b245b663a6a6d512f7ef50f47) I added some comments